### PR TITLE
Add missing sso-auth-modal

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix MANAGER-5262
| License          | BSD 3-Clause

## Description

- add `@ovh-ux/ng-translate-async-loader` dependency in `@ovh-ux/ng-ovh-sso-auth-modal-plugin`
- clean controller injection in `@ovh-ux/ng-ovh-sso-auth-modal-plugin`
- add  `@ovh-ux/ng-ovh-sso-auth-modal-plugin` in `@ovh-ux/manager-public-cloud`
- add  `@ovh-ux/ng-ovh-sso-auth-modal-plugin` in `@ovh-ux/manager-hub`

### CODE REVIEW

File: packages/manager/apps/hub/package.json [hunk 1]
```diff
@@ -53,6 +55,7 @@
     "angular-sanitize": "^1.7.8",
     "angular-translate": "^2.18.1",
     "angular-translate-loader-pluggable": "^1.3.1",
+    "angular-ui-bootstrap": "1.3.3",
```
Comment (marie-j) [inline packages/manager/apps/hub/package.json:58]: 😭

**Comment by JeremyDec** [comment]
@cbourgois there is conflict

**Comment by cbourgois** [comment]
> @cbourgois there is conflict

Rebased on `develop`

**Comment by marie-j** [comment]
Hello @cbourgois, 
Could you please resolve existing conflicts in order to merge this PR ? 
Moreover there is no internal ticket reference, is it wanted ? 
Thanks

**Comment by cbourgois** [comment]
> Hello @cbourgois,
> Could you please resolve existing conflicts in order to merge this PR ?
> Moreover there is no internal ticket reference, is it wanted ?
> Thanks

- Rebased on `develop`
- Issue reference added to `MANAGER-5262` in `feat` commits and PR description

Thanks.

**Review by marie-j — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Review by antleblanc — DISMISSED** [review]
Well done @cbourgois!

**thought:** It might impact the dist since these following dependencies are added
but could be tackle into a separated task if needed to determine what could be
expected for such a login functionality.

* `angular-ui-bootstrap`
* `angular-animate`
* `@ovh-ux/ng-ovh-sso-auth-modal-plugin`
*`ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css`

**Before:**
```sh
$ du -ch packages/manager/apps/hub/dist | grep total
6.7M	total
```

**After:**
```sh
$ du -ch packages/manager/apps/hub/dist/ | grep total
7.4M	total
```

**Comment by cbourgois** [comment]
Rebased on `develop`

**Review by Jisay — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**